modules/effects/spec/types/provide_effects.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { expecter } from 'ts-snippet';
+import { compilerOptions } from './utils';
+
+describe('provideEffects()', () => {
+  const expectSnippet = expecter(
+    (code) => `
+      import { provideEffects } from '@ngrx/effects';
+
+      class EffectFixture {}
+
+      ${code}`,
+    compilerOptions()
+  );
+
+  it('should compile without params', () => {
+    expectSnippet(`
+        provideEffects();
```

Comment:
Same question as above

Reply:
To keep the API the same.
I also don't know if it would be possible to throw here because of the spreaded array syntax (it would be an empty array in this case)